### PR TITLE
Tweak documentation

### PR DESCRIPTION
**Pull request by homeworkprod**
## Change Summary

I found a few spots in the documentation to impact the reading flow a bit, so I changed them.

## Related issue number

none

## Checklist

* n/a – Unit tests for the changes exist
* n/a – Tests pass on CI and coverage remains at 100%
* n/a – Documentation reflects the changes where applicable
* [x] `changes/<pull request or issue id>-<github username>.md` file added describing change
  (see [changes/README.md](https://github.com/samuelcolvin/pydantic/blob/master/changes/README.md) for details)
* [x] My PR is ready to review, **please add a comment including the phrase "please review" to assign reviewers**

### CODE REVIEW

**Comment by homeworkprod** [comment]
please review

File: changes/4143-homeworkprod.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Tweak documentation: Fix command to install python-dotenv, align spelling
```
Comment (hramezani) [inline changes/4143-homeworkprod.md:1]: Do we really need to mention this in changes?

Comment (homeworkprod) [reply changes/4143-homeworkprod.md:1]: I'm fine with not having this in the changelog.

I just added the file because the PR template requested me to.

Comment (hramezani) [reply changes/4143-homeworkprod.md:1]: I think we can remove this. please update the PR. Thanks!

**Comment by hramezani** [comment]
please update

**Comment by hramezani** [comment]
please update

File: changes/4143-homeworkprod.md (hunk 1 above)
Comment (homeworkprod) [reply changes/4143-homeworkprod.md]: Done.

**Comment by homeworkprod** [comment]
I have rebased, resolved conflicts, and removed the commit that adds the changes file.

**Comment by hramezani** [comment]
Thanks @homeworkprod

**Review by hramezani — APPROVED** [review]

**Pull request merged**